From 0775825c49f6261133c221475d100f92eeb4d8c1 Mon Sep 17 00:00:00 2001
From: KaKE-567 <kake@build.local>
Date: Wed, 26 Aug 2026 06:51:17 +0000
Subject: [PATCH] miatoll: Enable biometric Face Unlock feature

---
 device.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/device.mk b/device.mk
index 1c11e63..90f12c0 100644
--- a/device.mk
+++ b/device.mk
@@ -40,3 +40,8 @@ PRODUCT_COPY_FILES += \
 
 # Thermal
 PRODUCT_COPY_FILES += $(foreach f,$(wildcard $(LOCAL_PATH)/thermal/*.conf),$(f):$(TARGET_COPY_OUT_VENDOR)/etc/$(notdir $(f)))
+
+# Biometrics Face Unlock
+PRODUCT_COPY_FILES += \
+    frameworks/native/data/etc/android.hardware.biometrics.face.xml:$(TARGET_COPY_OUT_SYSTEM)/etc/permissions/android.hardware.biometrics.face.xml
+
-- 
2.43.0

