diff --git a/core/Makefile b/core/Makefile
index f6c2463..c6126dc 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -301,11 +301,7 @@ endif
 # non-default dev keys (usually private keys from a vendor directory).
 # Both of these tags will be removed and replaced with "release-keys"
 # when the target-files is signed in a post-build step.
-ifeq ($(DEFAULT_SYSTEM_DEV_CERTIFICATE),build/make/target/product/security/testkey)
-BUILD_KEYS := test-keys
-else
-BUILD_KEYS := dev-keys
-endif
+BUILD_KEYS := release-keys
 BUILD_VERSION_TAGS += $(BUILD_KEYS)
 BUILD_VERSION_TAGS := $(subst $(space),$(comma),$(sort $(BUILD_VERSION_TAGS)))
 
@@ -320,7 +316,7 @@ ifeq (,$(strip $(BUILD_FINGERPRINT)))
   else
     BF_BUILD_NUMBER := $(file <$(BUILD_NUMBER_FILE))
   endif
-  BUILD_FINGERPRINT := $(PRODUCT_BRAND)/$(TARGET_PRODUCT)/$(TARGET_DEVICE):$(PLATFORM_VERSION)/$(BUILD_ID)/$(BF_BUILD_NUMBER):$(TARGET_BUILD_VARIANT)/$(BUILD_VERSION_TAGS)
+  BUILD_FINGERPRINT := $(PRODUCT_BRAND)/$(TARGET_DEVICE)/$(TARGET_DEVICE):$(PLATFORM_VERSION)/$(BUILD_ID)/Zedex-1.0:user/release-keys
 endif
 # unset it for safety.
 BF_BUILD_NUMBER :=
@@ -355,20 +351,7 @@ OEM_THUMBPRINT_PROPERTIES := $(filter $(KNOWN_OEM_THUMBPRINT_PROPERTIES),\
     $(PRODUCT_OEM_PROPERTIES))
 
 # Display parameters shown under Settings -> About Phone
-ifeq ($(TARGET_BUILD_VARIANT),user)
-  # User builds should show:
-  # release build number or branch.buld_number non-release builds
-
-  # Dev. branches should have DISPLAY_BUILD_NUMBER set
-  ifeq (true,$(DISPLAY_BUILD_NUMBER))
-    BUILD_DISPLAY_ID := $(BUILD_ID).$(BUILD_NUMBER_FROM_FILE) $(BUILD_KEYS)
-  else
-    BUILD_DISPLAY_ID := $(BUILD_ID) $(BUILD_KEYS)
-  endif
-else
-  # Non-user builds should show detailed build information
-  BUILD_DISPLAY_ID := $(build_desc)
-endif
+BUILD_DISPLAY_ID := $(LINEAGE_VERSION)
 
 # Accepts a whitespace separated list of product locales such as
 # (en_US en_AU en_GB...) and returns the first locale in the list with
diff --git a/target/product/aosp_base_telephony.mk b/target/product/aosp_base_telephony.mk
index 0d4e333..f326a00 100644
--- a/target/product/aosp_base_telephony.mk
+++ b/target/product/aosp_base_telephony.mk
@@ -14,6 +14,3 @@
 # limitations under the License.
 #
 $(call inherit-product, $(SRC_TARGET_DIR)/product/full_base_telephony.mk)
-
-PRODUCT_PACKAGES += \
-    messaging
diff --git a/target/product/handheld_product.mk b/target/product/handheld_product.mk
index a4b44c4..bcd9d94 100644
--- a/target/product/handheld_product.mk
+++ b/target/product/handheld_product.mk
@@ -22,14 +22,10 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/media_product.mk)
 
 # /product packages
 PRODUCT_PACKAGES += \
-    Browser2 \
-    Calendar \
     Camera2 \
-    Contacts \
     DeskClock \
     Gallery2 \
     Launcher3QuickStep \
-    Music \
     OneTimeInitializer \
     Provision \
     QuickSearchBox \
diff --git a/tools/releasetools/common.py b/tools/releasetools/common.py
index 71679e8..30a0b14 100644
--- a/tools/releasetools/common.py
+++ b/tools/releasetools/common.py
@@ -1600,7 +1600,7 @@ def ZipWriteStr(zip_file, zinfo_or_arcname, data, perms=None,
 
   if not isinstance(zinfo_or_arcname, zipfile.ZipInfo):
     zinfo = zipfile.ZipInfo(filename=zinfo_or_arcname)
-    zinfo.compress_type = zip_file.compression
+    zinfo.compress_type = compress_type if compress_type is not None else zipfile.ZIP_DEFLATED
     if perms is None:
       perms = 0o100644
   else:
@@ -2148,10 +2148,10 @@ class BlockDifference(object):
     #   decompression_time: 15s  | 25s                | 25s
 
     if not self.src:
-      brotli_cmd = ['brotli', '--quality=6',
+      brotli_cmd = ['brotli', '--quality=8',
                     '--output={}.new.dat.br'.format(self.path),
                     '{}.new.dat'.format(self.path)]
-      print("Compressing {}.new.dat with brotli".format(self.partition))
+      print("Compressing {}.new.dat with brotli (quality 8)".format(self.partition))
       RunAndCheckOutput(brotli_cmd)
 
       new_data_name = '{}.new.dat.br'.format(self.partition)
diff --git a/tools/releasetools/ota_from_target_files.py b/tools/releasetools/ota_from_target_files.py
index 0cff60f..588bdbf 100755
--- a/tools/releasetools/ota_from_target_files.py
+++ b/tools/releasetools/ota_from_target_files.py
@@ -1055,7 +1055,7 @@ else if get_stage("%(bcb_dev)s") == "3/3" then
     script.Comment("Stage 3/3")
 
   # Dump fingerprints
-  script.Print("Target: {}".format(target_info.fingerprint))
+  script.Print("Target: Zedex-1.0-OFFICIAL-miatoll:user/release-keys")
 
   device_specific.FullOTA_InstallBegin()
 
@@ -1733,7 +1733,7 @@ else if get_stage("%(bcb_dev)s") != "3/3" then
 
   # Dump fingerprints
   script.Print("Source: {}".format(source_info.fingerprint))
-  script.Print("Target: {}".format(target_info.fingerprint))
+  script.Print("Target: Zedex-1.0-OFFICIAL-miatoll:user/release-keys")
 
   script.Print("Verifying current system...")
 
